cli/commands/build.go

```diff
@@ -0,0 +1,38 @@
+/*
+Creation of build command. Details on what is executed is described in 'Long'
+field of command variable.
+*/
+
+package commands
+
+import (
+	"fmt"
+	"os/exec"
+	"github.com/spf13/cobra"
+)
+
+var buildCmd = &cobra.Command{
+	Use: "build",
+	Short: "Builds executable for app",
+	Long: "Completes any compilation needed, builds executable with 'run' as name",
+	Args: cobra.ExactArgs(0),
+	Run: func(cmd *cobra.Command, args[]string) {
+		app := "go"
+		arg0 := "build"
+		arg1 := "-o"
+		arg2 := "run"
+
+		exec_output := exec.Command(app, arg0, arg1, arg2)
+		stdout, err := exec_output.Output()
+		
+		if err != nil {
+			fmt.Println(err.Error())
```

Comment:
Just an idea, but it might be nice for all of us to add a string before any of our print messages to specify what component of the system it is in. So your all of your print messages you could add `"CLI: "` beforehand. Thoughts?

Reply:
I agree it would be helpful. Later we can decide on whether to keep it in the final version or not.